app/components/Sidebar.tsx

```diff
@@ -1,7 +1,48 @@
+'use client';
+
+import { useState } from "react";
+
 export function Sidebar() {
+  const [activeTab, setActiveTab] = useState("algoritmos");
```

Comment:
What do you think in work with types here?

This `useState` could be typed like:

```suggestion

type ActiveTabs = 'algorithms' | 'courses';

export function Sidebar() {
  const [activeTab, setActiveTab] = useState<ActiveTabs>("algorithms");
```

Reply:
Hadn't thought of it. I think that's a great idea